content/master/concepts/pods.md

```diff
@@ -0,0 +1,231 @@
+permissions for the newly created API endpoints. 
+
+The RBAC manager pod manages these functions. 
+
+The RBAC manager pod supports two policy options:
+* `All` - The default policy, configures and manages all RBAC capabilities.
+* `Basic` - Only manage policies related to Composite resources. 
+
+Change from the default `All` to the `Basic` policy with the `-m Basic` pod 
+argument.
+
+Disable the RBAC manager after installation by deleting the
+`crossplane-rbac-manager` deployment from the `crossplane-system` namespace.
+
+Disable the RBAC manager before installation by editing the Helm `values.yaml`
+file, setting `rbacManager.deploy` to `false`.
+
+{{< hint "note" >}}
+
+Instructions for changing Crossplane pod settings during installation are in the
+[Crossplane Install]({{<ref "../software/install">}}) section. 
+{{< /hint >}}
+
+## Leader election
+
+By default only a single Crossplane pod runs in a cluster. If more than one
+Crossplane pod runs both pods try to manage Crossplane resources. To prevent
+conflicts Crossplane uses a _leader election_ to have a single pod in control at
+a time. Other Crossplane pods standby until the leader fails. 
+
+{{<hint "important" >}}
+Leader election applies to both the Crossplane pod and the RBAC manager pod.
+{{< /hint >}}
+
+Enable leader elections with the `--leader-election` pod argument.
+
+{{< hint "warning" >}}
+<!-- vale write-good.TooWordy = NO -->
+<!-- "multiple" -->
+<!-- vale write-good.Passive = NO -->
+<!-- allow "is unsupported" --> 
+Running multiple Crossplane pods without leader election is unsupported.
+<!-- vale write-good.Passive = YES -->
+<!-- vale write-good.TooWordy = YES -->
+{{< /hint >}}
+
+
+## Change pod settings
+
+Changing Crossplane pod settings either before installing Crossplane by editing
+the Helm `values.yml` file or after installation by editing the `Deployment`.
+
+The full list of configuration options are available in the 
+[Crossplane Install]({{<ref "../software/install">}}) section. 
+
+{{< hint "note" >}}
+
+Instructions for changing Crossplane pod settings during installation are in the
+[Crossplane Install]({{<ref "../software/install">}}) section. 
+{{< /hint >}}
+
+### Edit the deployment
+{{< hint "note" >}}
+These settings apply to both the `crossplane` and `rbac-manager` pods and
+`Deployments`.
+{{< /hint >}}
+
+To change the settings of an installed Crossplane pod, edit the `crossplane`
+deployment in the `crossplane-system` namespace with the command
+
+`kubectl edit deployment crossplane --namespace crossplane-system`
+
+{{< hint "warning" >}}
+Updating the Crossplane deployment restarts the Crossplane pod.
+{{< /hint >}}
+
+Add Crossplane pod arguments to the 
+{{<hover label="args" line="7" >}}spec.template.spec.containers[].args{{< /hover >}}
+section of the deployment.
+
+For example, to change the `sync-interval` add 
+{{<hover label="args" line="10" >}}--sync-interval=30m{{< /hover >}}.
```

Comment:
Line="10" doesn't seem like the right line here.

Reply:
fixed